# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rustc/riscv.patch
# Copyright (C) 2021 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rustc-1.53.0-src/compiler/rustc_target/src/spec/mod.rs.vanilla	2021-07-09 13:33:36.511141889 +0200
+++ rustc-1.53.0-src/compiler/rustc_target/src/spec/mod.rs	2021-07-09 13:35:32.915147894 +0200
@@ -866,10 +866,14 @@
     ("riscv32imc-unknown-none-elf", riscv32imc_unknown_none_elf),
     ("riscv32imac-unknown-none-elf", riscv32imac_unknown_none_elf),
     ("riscv32gc-unknown-linux-gnu", riscv32gc_unknown_linux_gnu),
+    ("riscv32-unknown-linux-gnu", riscv32_unknown_linux_gnu),
     ("riscv32gc-unknown-linux-musl", riscv32gc_unknown_linux_musl),
+    ("riscv32-unknown-linux-musl", riscv32_unknown_linux_musl),
     ("riscv64imac-unknown-none-elf", riscv64imac_unknown_none_elf),
     ("riscv64gc-unknown-none-elf", riscv64gc_unknown_none_elf),
     ("riscv64gc-unknown-linux-gnu", riscv64gc_unknown_linux_gnu),
+    ("riscv64-unknown-linux-gnu", riscv64_unknown_linux_gnu),
     ("riscv64gc-unknown-linux-musl", riscv64gc_unknown_linux_musl),
+    ("riscv64-unknown-linux-musl", riscv64_unknown_linux_musl),
 
     ("aarch64-unknown-none", aarch64_unknown_none),

--- rustc-1.62.1-src/vendor/cc-1.0.69/src/lib.rs.vanilla	2022-08-10 14:56:01.639529235 +0200
+++ rustc-1.62.1-src/vendor/cc-1.0.69/src/lib.rs	2022-08-10 14:56:26.321528251 +0200
@@ -2621,6 +2621,10 @@
             "riscv32gc-unknown-linux-gnu" => Some("riscv32-linux-gnu"),
             "riscv64gc-unknown-linux-musl" => Some("riscv64-linux-musl"),
             "riscv32gc-unknown-linux-musl" => Some("riscv32-linux-musl"),
+            "riscv64-unknown-linux-gnu" => Some("riscv64-linux-gnu"),
+            "riscv32-unknown-linux-gnu" => Some("riscv32-linux-gnu"),
+            "riscv64-unknown-linux-musl" => Some("riscv64-linux-musl"),
+            "riscv32-unknown-linux-musl" => Some("riscv32-linux-musl"),
             "s390x-unknown-linux-gnu" => Some("s390x-linux-gnu"),
             "sparc-unknown-linux-gnu" => Some("sparc-linux-gnu"),
             "sparc64-unknown-linux-gnu" => Some("sparc64-linux-gnu"),
--- rustc-1.62.1-src/vendor/cc/src/lib.rs.vanilla	2022-08-10 14:56:01.639529235 +0200
+++ rustc-1.62.1-src/vendor/cc/src/lib.rs	2022-08-10 14:56:26.321528251 +0200
@@ -2621,6 +2621,10 @@
             "riscv32gc-unknown-linux-gnu" => Some("riscv32-linux-gnu"),
             "riscv64gc-unknown-linux-musl" => Some("riscv64-linux-musl"),
             "riscv32gc-unknown-linux-musl" => Some("riscv32-linux-musl"),
+            "riscv64-unknown-linux-gnu" => Some("riscv64-linux-gnu"),
+            "riscv32-unknown-linux-gnu" => Some("riscv32-linux-gnu"),
+            "riscv64-unknown-linux-musl" => Some("riscv64-linux-musl"),
+            "riscv32-unknown-linux-musl" => Some("riscv32-linux-musl"),
             "s390x-unknown-linux-gnu" => Some("s390x-linux-gnu"),
             "sparc-unknown-linux-gnu" => Some("sparc-linux-gnu"),
             "sparc64-unknown-linux-gnu" => Some("sparc64-linux-gnu"),

